packages/vue/src/components/templates/SfConfirmOrder/SfConfirmOrder.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+  <div class="sf-confirm-order">
+    <slot name="heading">
+      <SfHeading
+        :title="orderTitle"
+        :level="orderTitleLevel"
+        class="
+          sf-heading--left sf-heading--no-underline
+          sf-confirm-order__heading
+        "
+      />
+    </slot>
+    <slot name="table">
+      <SfTable class="sf-table--bordered sf-confirm-order__table">
+        <SfTableHeading class="sf-confirm-order__table-row">
+          <SfTableHeader
+            class="sf-confirm-order__table-header sf-confirm-order__table-image"
+            >Item</SfTableHeader
+          >
+          <SfTableHeader
+            v-for="tableHeader in tableHeaders"
+            :key="tableHeader"
+            class="sf-confirm-order__table-header"
+            :class="{
+              'sf-confirm-order__table-description':
+                tableHeader === 'Description',
+            }"
+            >{{ tableHeader }}
+          </SfTableHeader>
+        </SfTableHeading>
+        <SfTableRow
+          v-for="(product, index) in orderItems"
+          :key="index"
+          class="sf-confirm-order__table-row"
+          data-testid="product-table-row"
+        >
+          <SfTableData class="sf-confirm-order__table-image">
+            <SfImage
+              :src="product.image"
+              :alt="product.title"
+              data-testid="product-image-table-data"
+            />
+          </SfTableData>
+          <SfTableData class="sf-confirm-order__table-data"
+            >{{ product.configuration[0].value }}
+          </SfTableData>
+          <SfTableData
+            class="sf-confirm-order__table-description"
+            data-testid="product-description-table-data"
+          >
+            <div class="sf-confirm-order__product-title">
+              {{ product.title }}
+            </div>
+            <div class="sf-confirm-order__product-sku">{{ product.sku }}</div>
+          </SfTableData>
+          <SfTableData class="sf-confirm-order__table-data">{{
+            product.qty
+          }}</SfTableData>
+
+          <SfTableData class="sf-confirm-order__table-data"
+            >{{ product.configuration[1].value }}
+          </SfTableData>
+          <SfTableData class="sf-confirm-order__table-data">
+            <SfPrice
+              :regular="product.price.regular"
+              :special="product.price.special"
+              class="sf-confirm-order__product-price"
+            />
+          </SfTableData>
+        </SfTableRow>
+      </SfTable>
+    </slot>
+    <div class="sf-confirm-order__totals">
+      <slot name="summary">
+        <SfProperty
+          name="Subtotal"
```

Comment:
it should also be possible to change this name in props, wdyt?

Reply:
Add to issue with customization of templates